storybook/src/elements/nuxeo-collapsible/nuxeo-collapsible.stories.js

```diff
@@ -0,0 +1,73 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import { boolean, select, text } from '@storybook/addon-knobs';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-collapsible';
+
+const stories = storiesOf('UI/nuxeo-collapsible', module);
+
+stories
+  .add('Default', () => {
+    const heading = text('Heading', 'Nuxeo');
+    const chevronPosition = select('Chevron position', { start: 'start', end: 'end' }, 'end');
+    const animation = boolean('Animation', true);
+    const expanded = boolean('Expanded', false);
+
+    return html`
+      <style>
+        :root {
+          display: block;
+          width: 300px;
+          margin: 2rem;
+        }
+
+        .heading {
+          font-weight: bold;
+        }
+
+        p {
+          margin: 0;
+        }
+      </style>
+      <nuxeo-collapsible .chevronPosition="${chevronPosition}" ?animation="${animation}" ?expanded="${expanded}">
+        <span class="heading" slot="heading">${heading}</span>
+        <p>Build Smarter Solutions for Today's Content Challenges</p>
+      </nuxeo-collapsible>
+    `;
+  })
+
+  .add('Nested collapsible', () => {
```

Comment:
this story isn't much different than the default, do we need it?

Reply:
I was trying to mimic an actual situation we will probably see, a nested collapsible (i.e., a collapsible inside another collapsible) but without the typical chevron. The idea here is to demo that the element can take several forms and shapes.